config/exchanges/AR-NEA_BR-S.yaml

```diff
@@ -0,0 +1,6 @@
+lonlat:
+  - -56.407453
+  - -28.880006
+parsers:
+  exchange: BR.fetch_exchange
+rotation: 110
```

Comment:
The `BR.py` parser does not support the `AR-NEA->BR-S` exchange so this will have to be implemented in the `AR.py` parser. I would suggest adding it in #4893 and then we can merge that first.

Reply:
Added in: f65e0b9